article/article.md

```diff
@@ -0,0 +1,248 @@
+# Live sync in React Native with CouchDb
+
+## Intro
+
+In this tutorial I will show you how to create a live game with data updating on multiple devices.
+
+Let's start with something fun and basic! A game in which two people will be able to play together online. Let's make a live `Rock - Paper - Scissors` game. I've already programmed it [here](https://github.com/jcalixte/rnrps) so we can focus straight away on the cool stuff 😄.
+
+This is a perfect opportunity to see how to build live data syncs in a mobile app with CouchDb. You'll of course be able to exploit these two technologies in many more use cases but this is a good way to start.
+
+Thanks to CouchDb we don't need to build any backend! By the end we'll have a React Native app connected to a local CouchDb database. This basic game will help us concentrate on the essentials parts: the live sync & update on our React component. Feel free to explore the components to understand how it's displayed. Let's start!
+
+## 1. Overview
+
+### Rock-Paper-Scissors
+
+What exactly will our online game be able to do?
+
+The famous and popular Rock-Paper-Scissors will be played by two people who will join the game thanks to its id. Other users will also be able to participate and join as spectators. Afer each round, the app will update the game and at the end display the final score.
+
+Here a quick demo of what a player will see.
+
+![Demo](./assets/rps-demo.gif)
+
+First steps are to clone the [repo](https://github.com/jcalixte/rnrps) and simply run the command `yarn`.
+
+### What is CouchDb
+
+Before getting deeper into the app, it seems primordial to first know the technology behind it.
```

Comment:
"primordial" c'est un peu bizarre je trouve, ça fait pas très idiomatique ... je dirais un truc du genre :
"Before getting deeper into the app, let's dive into the technology behind it"